parm/config/config.fcst

```diff
@@ -35,6 +35,11 @@ if [ $QUILTING = ".true." ]; then
     export npe_fcst_gfs=$(echo " $npe_fcst_gfs  + $WRITE_GROUP_GFS * $WRTTASK_PER_GROUP_GFS" | bc)
 fi
 
+if [ $DO_WAVE = "YES" ] ; then
+  export npe_fcst=$((npe_fcst + npe_wav))
+  export npe_fcst_gfs=$((npe_fcst_gfs + npe_wav))
+fi
+
```

Comment:
OK, now I see the reason why ncep_fcst and npe_fcst_gfs are used in env/env.* files. 
But this block cloud have been moved up after line 21 "export FCSTEXEC="global_fv3gfs.x"

Reply:
I put the addition of the wave npes after the QUILTING if-block because we needed npe_fv3 to not include the wave npes at this point and only include the atmosphere (fv3) resources. Then we add npe_wav to npe_fcst[_gfs] which is what is used with $launcher at runtime. This feeds into the adjustment made to the env files to not use npe_fv3 but npe_fcst[_gfs] instead with the $launcher commands. With additional components coming in the future this seemed the time to split npe_fv3 (atmos) from the final npe_fcst[_gfs] variables (all components).

The changes to exglobal_fcst_nemsfv3gfs.sh include the following npe_fv3/NTASKS variables:

```
NTASKS_FV3=${NTASKS_FV3:-$npe_fv3}

NTASKS_FV3m1=$((NTASKS_FV3-1))
atm_petlist_bounds=" 0 $((NTASKS_FV3-1))"
wav_petlist_bounds=" $((NTASKS_FV3)) $((NTASKS_FV3m1+npe_wav))"

# Set NTASKS_CFG to reflect cplwav
NTASKS_CFG=$NTASKS_FV3
if [ $cplwav = ".true." ]; then
  NTASKS_CFG=$((NTASKS_FV3 + npe_wav))
fi
```

This aspect of adding the waves gave us some pause so I'm happy to discuss this change further since this is an important change.